src/test/java/com/google/cloud/tools/managedcloudsdk/install/DownloaderTest.java

```diff
@@ -156,29 +152,26 @@ public void testDownload_failIfExists() throws IOException, InterruptedException
 
   @Test
   public void testDownload_interruptTriggersCleanup() throws IOException, InterruptedException {
-    final Path destination = tmp.getRoot().toPath().resolve("destination-file");
+    Path destination = tmp.getRoot().toPath().resolve("destination-file");
     long testFileSize = Downloader.BUFFER_SIZE * 10 + 1;
     Path testSourceFile = createTestRemoteResource(testFileSize);
-    final URL fakeRemoteResource = testSourceFile.toUri().toURL();
+    URL fakeRemoteResource = testSourceFile.toUri().toURL();
 
     // Start a new thread for this test to avoid mucking with Thread state when
     // junit reuses threads.
     Thread testThreadToInterrupt =
         new Thread(
-            new Runnable() {
-              @Override
-              public void run() {
-                Downloader downloader =
-                    new Downloader(fakeRemoteResource, destination, null, mockProgressListener);
-                Thread.currentThread().interrupt();
-                try {
-                  downloader.download();
-                  Assert.fail("InterruptedException expected but not thrown.");
-                } catch (InterruptedException ex) {
-                  Assert.assertEquals("Download was interrupted", ex.getMessage());
-                } catch (IOException e) {
-                  Assert.fail("Test failed due to IOException");
-                }
+            () -> {
```

Comment:
don't do this. JavaNCSS can't handle it.

Reply:
That's very unfortunate.